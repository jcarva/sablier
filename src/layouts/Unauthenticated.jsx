/* @flow */
import { Outlet } from 'react-router-dom';
import { LayoutWrapper, ContentWrapper } from './common';
import type { Component } from '../types';

const Unauthenticated = (): Component => {
  return (
    <LayoutWrapper>
      <ContentWrapper>
        <Outlet />
      </ContentWrapper>
    </LayoutWrapper>
  );
};

export default Unauthenticated;
