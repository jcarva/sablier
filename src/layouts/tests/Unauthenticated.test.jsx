/* @flow */
import renderer from 'react-test-renderer';
import Unauthenticated from '../Unauthenticated';

describe('Unauthenticated', () => {
  it('[snapshot]', () => {
    expect(renderer.create(<Unauthenticated />).toJSON()).toMatchSnapshot();
  });
});
