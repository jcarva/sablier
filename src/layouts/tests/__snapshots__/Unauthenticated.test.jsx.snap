// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unauthenticated [snapshot] 1`] = `
<div>
  <div
    className="header"
  >
    Header
  </div>
  <div />
</div>
`;
